package.json

```diff
@@ -82,7 +85,7 @@
     "sql.js": "pastelnetwork/sql.js#230351f",
     "squoosh": "pastelnetwork/squoosh#d44b1b4",
     "styled-components": "5.3.0",
-    "tailwindcss": "2.1.2",
+    "tailwindcss": "npm:@tailwindcss/postcss7-compat@^2.0.4",
```

Comment:
Why is this needed?

Reply:
Storybook fails with that version of tailwind.
I will update it when the issue would be patched by storybook version upgrade later.

Reply:
It's resolved now. Check [here](https://github.com/pastelnetwork/pastel-electron-wallet/pull/95/commits/fec4e35e649917c94ad6cfaea1bc387111258a22) plz.